pkgs/development/python-modules/setuptools/44.0.nix

```diff
@@ -27,14 +27,23 @@ let
       name = "${pname}-${version}-source";
     };
 
+    patches = [
+      ./tag-date.patch
+    ];
+
     buildPhase = ''
       ${python.pythonForBuild.interpreter} bootstrap.py
       ${python.pythonForBuild.interpreter} setup.py sdist --formats=gztar
+
+      # Here we untar the sdist and retar it in order to control the timestamps
+      # of all the files included
+      tar -xzf dist/${pname}-${version}.post0.tar.gz -C dist/
```

Comment:
Is post0 always the same?

Reply:
Yes, the name of the release tag `post0` is deterministic now (it used to contain the date). The patch of `setup.cfg` ensures that.

It is possible that an update of the source changes that (if the build system upstream is updated), but I would assume that the `tag-date.patch` would also need to be updated then. If/when that happens this `buildPhase` should fail loudly, so I don't think hardcoding it is an issue.

If you have any suggestion to make this better, do tell! :smile: